docs/conf.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+import os
+import sys
+from datetime import datetime
+
+
+sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(1, os.path.abspath('../src/twister2'))

```

Comment:
why do docs need this? shouldn't it be installed when building docs?

Reply:
Can be remove, I think so.